build/Build.csx

```diff
@@ -1,28 +1,48 @@
 #! "netcoreapp2.0"
-#load "nuget:Dotnet.Build, 0.2.9"
+#load "nuget:Dotnet.Build, 0.3.1"
```

Comment:
what changed here?

Reply:
Had to add the possibility to specify the target framework when doing `DotNet.Publish` since we now have multiple target frameworks. Otherwise nothing much to write home about 👍